locale/Makefile

```diff
@@ -42,18 +43,20 @@ uniq-po:
 	done
 
 tx-pull: $(EDITFILES)
-	tx pull -f
+	cd .. && tx pull -f
 	for f in $(EDITFILES) ; do \
 		sed -i 's/^\("Project-Id-Version: \).*$$/\1$(DOMAIN) $(VERSION)\\n"/' $$f; \
 	done
 
 tx-update: tx-pull
 	@echo
-	@echo Run rake plugin:gettext[$(DOMAIN)] from the Foreman installation, then make -C locale mo-files to finish
+	@echo Run rake plugin:gettext[$(DOMAIN)] from the Foreman installation
```

Comment:
So the correct steps for the full plugin translations update are as follows:
1. `make -C locale tx-pull` (In plugin dir)
2. `rake plugin:gettext[$(DOMAIN)]` from the Foreman installation
3. `rake plugin:po_to_json[$(DOMAIN)]` from the Foreman installation
4. `make -C locale mo-files` from plugin dir

Nothing else is needed?

Reply:
@ekohl can you confirm the steps, I just to be sure before I go over the other 15 PRs and do the changes ...

Reply:
It has already been accepted into the plugin template

Reply:
Nice

Reply:
@stejskalleos indeed, those are the current steps

@adamruzicka I have been thinking that perhaps we should have added it to `plugin:gettext[$DOMAIN]`.

Reply:
I tried that at first, but I couldn't get it to work properly. Otoh it was at the time before my patches were accepted to potojson and gettext_i18n_rails_js . Maybe now it could be made to work?

Reply:
I was confused by Foreman, where I thought it happened automatically but it's actually part of the `update-po` task in `locale/Makefile`. So I think you're right that core and plugins are now at the same level.

We could tell them to run `rake plugin:gettext[$(DOMAIN)] plugin:po_to_json[$(DOMAIN)]` to simplify things.

Also, there's this in Foreman core: https://github.com/theforeman/foreman/blob/f7e5525dc8a3af7dd5333038f633f4e8b462f5be/lib/tasks/locale.rake#L45-L46